career-readiness.md

```diff
@@ -6,16 +6,16 @@ The LEARN program focuses on readying our students for a career in technology. T
 
 Deploying a project is an exciting, yet often challenging process. There are usually small errors that cause deployments to fail or cause bugs in production. One solution to this is to deploy early and update the deployment as the code evolves.
 
-Submit the link to your deployed portfolio site:
+Submit the link to your deployed portfolio site: https://michael-reyes-portfolio.onrender.com
 
 We recommend GitHub pages for HTML/CSS and Render.com for React. Your portfolio site should display AT LEAST "Hello World" text. Completely blank sites will be considered incomplete. 
 
 ### LinkedIn
 
-Your LinkedIn url:
+Your LinkedIn url: https://github.com/learn-academy-2023-golf/week-5-assessment-MichaelReyes1.git
```

Comment:
This is the assessment's github repo link.  Please provide your linkedin profile url.

Reply:
rookie mistake sorry just repushed it